test/base/AppWithDefaultMockFixtures.scala

```diff
@@ -83,7 +83,8 @@ trait AppWithDefaultMockFixtures extends BeforeAndAfterEach with GuiceOneAppPerS
   protected def setNoExistingUserAnswers(): Unit =
     when(mockDataRetrievalActionProvider.apply(any())) thenReturn new FakeDataRetrievalAction(None)
 
-  protected def checkArrivalStatus(): Unit = {}
+  // TODO - delete?
+  protected def checkArrivalStatus(): Unit = ()
```

Comment:
I will look into this, think it comes from one of the tech debt tickets unsure why it does nothing though

Reply:
@Reece-Carruthers it looks to be a hangover from P4 - https://github.com/hmrc/declare-transit-movement-unloading-frontend/blob/3af44828a1710047fe39ec5c0b363eb3cf32a057/test/base/AppWithDefaultMockFixtures.scala#L65-L73. The mock was removed such that the method itself didn't do anything. So now we have a method in a lot of controller specs that isn't actually doing anything.